server/controllers/notes.js

```diff
@@ -37,10 +37,19 @@ const notesController = {
         Note.find({ deleted: false })
             .exec()
             .then(notes => {
-                return res.status(200).json({
-                    error: false,
-                    notes: notes
-                })
+                if(notes){
+                    return res.status(200).json({
+                        error: false,
+                        notes: notes,
+                        message:"Records found!"
+                    })
+                }else{
+                    return res.status(200).json({
+                        error: false,
+                        notes: {},
+                        message:"No records found!"
+                    })
```

Comment:
Can we do:
```js
notes = notes || {};

return res.status(200).json({
  error: false,
  notes: notes,
});
```

I don't think message is necessary in 200 status code and only required in error status codes.

Also, if the linter gives a warning that you can not reassign a param value, rename the param as `notesRecords` and use a new constant called `notes`.

```js
.then((notesRecords) => {
  const notes = notesRecords || {};
})
```

Reply:
ok

Reply:
it's not giving any error i reassigned to it